docs/user-guide/system-admin/configuration/document-storage-directory.md

```diff
@@ -1,31 +1,27 @@
+Zanata will store files for some uploaded documents on the file system in a configured directory. This directory must be configured by using a system property. Zanata will create the directory if it does not yet exist.
 
-Since files were previously stored as BLOBs in the database, there is a migration operation to move the files from the database to the file system. Migration is performed when liquibase is run, which will happen automatically at server startup, or can be initiated from the command line. If liquibase is run from the command line, the directory must be passed as a parameter to liquibase, in addition to the JNDI name binding.
+Since files were previously stored as BLOBs in the database, there is a migration operation to move the files from the database to the file system. Migration is performed when liquibase is run, which will happen automatically at server startup, or can be initiated from the command line. If liquibase is run from the command line, the directory must be passed as a parameter to liquibase, in addition to the system property.
 
 ## Configuring the Directory (JNDI)
-The path for document storage should be bound to JNDI name ```java:global/zanata/files/document-storage-directory```
+The path for document storage should be set using a system property when starting JBoss.
 
 ### Detailed Procedure
-Open `$JBOSS_HOME/standalone/configuration/standalone.xml` and look for the `naming` subsystem. Add the following section to the `bindings` section:
+Open `$JBOSS_HOME/standalone/configuration/standalone.xml` and look for the `system-properties` section. Add the following property declaration:
 
 ```xml
-<subsystem xmlns="urn:jboss:domain:naming:1.3">
-    ...
-    <bindings>
-        <simple name="java:global/zanata/files/document-storage-directory" value="/example/path"/>
-    </bindings>
-    ...
-</subsystem>
+<property name="zanata.files.documentstorage.directory" value="/example/path"/>
 ```
 
+Alternatively, you can pass this and other system properties to JBoss when starting it (see JBoss documentation for details on how to do this).
+
 ## Running Liquibase from command line
-This section will not explain how to run liquibase from the command line, just how to add the required parameter. The parameter is ```document.storage.directory``` and should be added to the liquibase command in the form ```-Ddocument.storage.directory=/example/path```. The value passed to this parameter should exactly match the value bound to the above JNDI name.
+This section will not explain how to run liquibase from the command line, just how to add the required parameter. The parameter is ```document.storage.directory``` and should be added to the liquibase command in the form ```-Ddocument.storage.directory=/example/path```. The value passed to this parameter should exactly match the value bound to the system property above.
```

Comment:
We need to be careful when changing the liquibase parameter. Liquibase may treat the changeset as modified, just because we stop passing a parameter to it, or because the parameter value changes. See https://github.com/zanata/zanata-server/blob/server-3.9.0-rc-1/zanata-war/src/main/resources/db/changelogs/db.changelog-3.1.xml#L265

I suppose that might be an argument for sticking to `document.storage.directory`.